src/platform/cannonlake/include/platform/lib/memory.h

```diff
@@ -261,7 +261,7 @@
 #define HEAP_BUFFER_BLOCK_SIZE		0x180
 #define HEAP_BUFFER_COUNT	(HEAP_BUFFER_SIZE / HEAP_BUFFER_BLOCK_SIZE)
 
-#define HEAP_SYSTEM_M_SIZE		0x8000 /* heap master core size */
+#define HEAP_SYSTEM_M_SIZE		0x9000 /* heap master core size */
 #define HEAP_SYSTEM_S_SIZE		0x6000 /* heap slave core size */
```

Comment:
@abonislawski: I would like this kind of commits to have a better commit message with a proper explanation. We cannot afford to play with magic numbers without explanations.

Also, please explain why do we need to change HEAP_SYSTEM_M_SIZE

Reply:
@abonislawski ping

Reply:
@dbaluta its obvious I think, there is no enough space so it needs to be increased

Reply:
@abonislawski  not enough space for what?

The commit message says  "ipc: increase msg max size" so I'm confused. Please do a commit message that explains

1) Why do we need to increase msg max size? What is the problem? What is a scenario for for the current msg max size doesn't work.

2) If changing HEAP_SYSTEM_M_SIZE is related to 1) please explain how? If not do the change in another commit and explain why it is needed.

Reply:
@dbaluta not enough space for increased msg size :D 
Well, ok, please check updated commit message.